operations/helm/charts/mimir-distributed/templates/smoke-test/smoke-test-job.yaml

```
@@ -52,6 +52,10 @@ spec:
             {{- range $key, $value := .Values.smoke_test.extraArgs }}
             - -{{ $key }}={{ $value }}
             {{- end }}
+          {{- if .Values.smoke_test.resources }}
+          resources:
+            {{ toYaml .Values.smoke_test.resources | nindent 12 }}
+          {{-end }}
```

The CI fails, with a notice about a broken template:

```suggestion
          {{- end }}
```

---

Issue fixed